content/utilities/tiny-leaderboard/plugin.js

```diff
@@ -0,0 +1,171 @@
+
+// ---------------------------------------------------------------
+
+// https://github.com/darkforest-eth/client/blob/19f2547dce33f3274afa86945e96fbfebc7693f5/src/Frontend/Views/Leaderboard.tsx#L119
+const roundEndTimestamp = '2021-07-07T21:00:00-07:00';
+const roundEndTime = new Date(roundEndTimestamp).getTime();
+
+function formatDuration(durationMs) {
+	if (durationMs < 0) return '';
+	const hours = Math.floor(durationMs / 1000 / 60 / 60);
+	const minutes = Math.floor((durationMs - hours * 60 * 60 * 1000) / 1000 / 60);
+	const seconds = Math.floor((durationMs - hours * 60 * 60 * 1000 - minutes * 60 * 1000) / 1000);
+	return (
+		timestampSection(hours) + ':' + timestampSection(minutes) + ':' + timestampSection(seconds)
+	);
+}
+
+function timestampSection(value) {
+	return value.toString().padStart(2, '0');
+}
+
+function getStrTimeUntilRoundEnds() {
+	const timeUntilEnd = roundEndTime - new Date().getTime();
+	if (timeUntilEnd <= 0) return "00:00";
+	else return formatDuration(timeUntilEnd);
+}
+
+// ---------------------------------------------------------------
+
+function formatNumber(num, decimalCount = 1) {
+	num = parseInt(num);
+	if (num < 1000) return num;
+	if (num < 1000000) return (num / 1000).toFixed(decimalCount) + "k";
+	if (num < 1000000000) return (num / 1000000).toFixed(decimalCount) + "m";
+	if (num < 1000000000000) return (num / 1000000000).toFixed(decimalCount) + "b";
+	return (num / 1000000000000, 2).toFixed(decimalCount) + "t";
+}
+
+async function downloadLeaderboard() {
+	return fetch('https://api.zkga.me/leaderboard')
+		.then(response => response.json())
+}
+
+// return example: 'hsl(285,100%,70%)'
+function getPlayerColor(ethAddress) {
+	return df.getProcgenUtils().getPlayerColor(ethAddress);
+}
+
+function Plugin() {
+	var o = {};
+	o.div;
+	o.div_timer;
+	o.div_playerList;
+	o.updateTimerInterv = null;
+	o.updateLeaderboardInterv = null;
+	o.firstRender = true;
+	o.leaderboard = [];
+
+	o.init = function () {
+		o.updateTimerInterv = setInterval(o.updateTimer, 1000);
+		o.updateInterv = setInterval(o.updateLeaderboard, 1000 * updateTimeInSeconds);
+		o.updateTimer();
+		o.updateLeaderboard();
+	}
+
+	o.render = function (container) {
+		o.container = container;
+		o.container.style.width = '250px';
+
+		o.div_timer = document.createElement('div');
+		o.div_timer.style.fontSize = "x-large";
+		o.div_timer.style.textAlign = "center";
+		o.div_timer.style["margin-top"] = "-10px";
+		container.appendChild(o.div_timer);
+
+		container.appendChild(document.createElement('hr'));
+
+		o.div_playerList = document.createElement('div');
+		o.div_playerList.style.width = o.container.offsetWidth+"px";
+		container.appendChild(o.div_playerList);
+
+		if (o.firstRender) {
+			o.init();
+			o.firstRender = false;
+		}
+	}
+
+	o.destroy = function () {
+		if (o.updateInterv !== null)
+			clearInterval(o.updateInterv);
+		if (o.updateTimerInterv !== null)
+			clearInterval(o.updateTimer);
```

Comment:
```suggestion
			clearInterval(o.updateTimerInterv);
```

Reply:
good catch :O

Reply:
you need to click commit suggestion if you like the change :)